debian/libnoopenh264-6.links

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/dh-exec
+[linux-amd64] /var/lib/flatpak/runtime/org.freedesktop.Platform.openh264/x86_64/2.0/active/files/extra/libopenh264.so.6 /usr/lib/${DEB_HOST_MULTIARCH}/openh264/extra/libopenh264.so.6
+[linux-arm64] /var/lib/flatpak/runtime/org.freedesktop.Platform.openh264/aarch64/2.0/active/files/extra/libopenh264.so.6 /usr/lib/${DEB_HOST_MULTIARCH}/openh264/extra/libopenh264.so.6
```

Comment:
The last time I checked this didn't actually exist for aarch64 – https://phabricator.endlessm.com/T30986 / https://gitlab.com/freedesktop-sdk/openh264-extension/-/issues/2#note_432603235 – but it seems harmless to have this here. And I think I checked back then and the soname is indeed 6 for the Cisco-provided aarch64 binary.

Reply:
Yes, my idea was that once it gets supported it would just work for us without extra changes.

Reply:
OK!